client/references.d.ts

```diff
@@ -0,0 +1 @@
+import "../server/src/db/zapatos/schema";
```

Comment:
Re filename: `references` isn't a great name as it is ambiguous about what is being referred to here - consider `schema`?

Reply:
`references.d.ts` feels slightly better to me over `schema.d.ts` since our client doesn't actually care about our database schema, it just happens to be that Zapatos declares its types in a `.d.ts` which can't be implicitly resolved like everything else--but fine changing if you think `schema.d.ts` is a better fit :)

Reply:
We have two schemas - storage and communication. This is the communication schema.

Reply:
sorry, I think there might be some confusion--`zapatos/schema.d.ts` are types generated directly from our Postgres database, not types that we manually define